packages/medusa/src/loaders/api.ts

```diff
@@ -12,7 +12,7 @@ type Options = {
 
 export default async ({ app, container, configModule }: Options) => {
   app.use(bodyParser.json())
-  app.use("/", routes(container, configModule.projectConfig))
+  app.use("/", routes(container, await Promise.resolve(configModule.projectConfig)))
```

Comment:
issue: Here the config should already be resolved and stored, we should not have any changes here

Reply:
done